### ISSUE TITLE
select: allow dynamic update items

### ISSUE THREAD

**Issue by aristov** [bem/bem-components#1100]

**Comment by narqo** [comment]
Close this now. See my [comment](https://github.com/bem/bem-components/pull/1163#issuecomment-68095469) in #1100 for details.

**Comment by awinogradov** [comment]
Please, reopen this issue! We really want this method!

**Comment by Guria** [comment]
:+1:

**Comment by voischev** [comment]
:+1:

**Comment by narqo** [comment]
> Please, reopen this issue! We really want this method

Does rerendering whole `select` is not suitable?

**Comment by awinogradov** [comment]
No, i want update items when select is open. Popup out of the select node ;(

**Comment by narqo** [comment]
> No, i want update items when select is open

So you have a `select` with some items checked. While it's opened you want to refill it with new list of items. During this process internal `button` and hidden controls will be refilled/recreated, `change` event will be fired and all other things connected with popup redrawing processes will be done. And in the end you will have completely new `select`, still opened and focused. 

Could you provide a more details of the need of such behaviour?

**Comment by Guria** [comment]
suggestion:

``` js
{
  block: 'select',
  tech: 'bemhtml',
  shouldDeps: {
    block: 'select',
    mod: 'dynamic',
    tech: 'js'
  }
}
```

**Comment by Guria** [comment]
maybe `tech: 'browser-bemhtml'` or something like that
